include/unifex/just_error.hpp

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2019-present Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <unifex/config.hpp>
+#include <unifex/receiver_concepts.hpp>
+#include <unifex/sender_concepts.hpp>
+#include <unifex/blocking.hpp>
+#include <unifex/std_concepts.hpp>
+
+#include <exception>
+#include <tuple>
+#include <type_traits>
+#include <utility>
+
+#include <unifex/detail/prologue.hpp>
+
+namespace unifex {
+namespace _just_error {
+
+template <typename Receiver, typename Error>
+struct _op {
+  struct type;
+};
+template <typename Receiver, typename Error>
+using operation = typename _op<remove_cvref_t<Receiver>, Error>::type;
+
+template <typename Receiver, typename Error>
+struct _op<Receiver, Error>::type {
+  UNIFEX_NO_UNIQUE_ADDRESS Error error_;
+  UNIFEX_NO_UNIQUE_ADDRESS Receiver receiver_;
+
+  void start() & noexcept {
+    unifex::set_error((Receiver &&) receiver_, (Error &&) error_);
+  }
+};
+
+template <typename Error>
+struct _sender {
+  class type;
+};
+template <typename Error>
+using sender = typename _sender<std::decay_t<Error>>::type;
+
+template <typename Error>
+class _sender<Error>::type {
+  UNIFEX_NO_UNIQUE_ADDRESS Error error_;
+
+  public:
+  template <
+      template <typename...> class Variant,
+      template <typename...> class Tuple>
+  using value_types = Variant<Tuple<>>;
```

Comment:
This should be `using value_types = Variant<>` to indicate that no set_value() call will be invoked.
This is currently indicating that it _may_ invoke `set_value(receiver)`